src/styles/mixins/buttons.scss

```diff
@@ -1,6 +1,8 @@
 @use "./focus";
 @use "../variables" as var;
 
+$button-padding: 0.5em;
```

Comment:
This is only used once and is probably not useful; artifact of an earlier approach.